src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.cs

```diff
@@ -312,29 +312,28 @@ public async Task When_Text_IsWrapping_Set()
 		{
 			var textbox = new TextBox();
 
-			var initialHeight = 500;
-
+			var initialHeight = 500;					
+			
 			textbox.Height = initialHeight;
-
-			var stackPanel = new StackPanel()
-			{
-				Children = { textbox }
-			};
-
-			WindowHelper.WindowContent = stackPanel;
+			
+			WindowHelper.WindowContent = textbox;
 
 			await WindowHelper.WaitForLoaded(textbox);		
 			
 			textbox.Text = "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremaaaaaaaaaaaaaaaaaa";
 			textbox.TextWrapping = TextWrapping.Wrap;
+			await WindowHelper.WaitForIdle();
 
-			Android.Graphics.Rect textBounds = new Android.Graphics.Rect();
+			var field = textbox.GetType().GetField("_textBoxView", System.Reflection.BindingFlags.NonPublic
+				| System.Reflection.BindingFlags.Instance);
 
-			textbox.GetHitRect(textBounds);
+			var textBoxView = field.GetValue(textbox);
 
-			await WindowHelper.WaitForIdle();
-			
-			Assert.IsTrue(textBounds.Height() <= initialHeight);						
+			var linecountProperty = textBoxView.GetType().GetProperty("LineCount");
```

Comment:
LineCount is not the right property, this will not be available on all platforms.

Get `ActualHeight` before and after and check that `ActualHeight` is greater. If it is, the test is successful.

Reply:
I did the change one more time and as I reported before ActualHeight never changes after wrapping and unwrapping the text. Still failing to assert condition.

![image](https://user-images.githubusercontent.com/5515535/155855719-699ef14c-6638-4646-88ed-291725a87474.png)

Reply:
It'd be best to display the values used in IsTrue, or to use IsGreater or similar. Are you reproducing the error on your machine?

Reply:
Yes, @jeromelaban I could reproduce this error in my machine. It was fixed and passes to the pipeline now!